sdks/java/extensions/sql/build.gradle

```diff
@@ -240,6 +240,12 @@ task integrationTest(type: Test) {
   useJUnit { }
 }
 
+task preCommit {
```

Comment:
I think we need something like https://github.com/apache/beam/blob/6751a1737c19d7022c30e850a718e73cd1909e30/build.gradle.kts#L731 here

Reply:
good point, will add

Reply:
added to java:extensions:sql:preCommit and verified that 
```
./gradlew :sdks:java:extensions:sql:preCommit -PtestJavaVersion=17 -Pjava17Home=.../java/17.0.8-tem/
```

produces log
```
> Task :sdks:java:testing:test-utils:verifyJavaVersion17
Java verified
```

Reply:
Thanks - I think we can now remove https://github.com/apache/beam/blob/6751a1737c19d7022c30e850a718e73cd1909e30/build.gradle.kts#L731 as well